pkgs/development/python-modules/debian-inspector/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, chardet
+, attrs
+, commoncode
+, pytestCheckHook
+, pythonAtLeast
+}:
+buildPythonPackage rec {
+  pname = "debian-inspector";
+  version = "0.9.10";
+
+  src = fetchPypi {
+    pname = "debian_inspector";
+    inherit version;
+    sha256 = "fd29a02b925a4de0d7bb00c29bb05f19715a304bc10ef7b9ad06a93893dc3a8c";
+  };
+
+  dontConfigure = true;
+
+  propagatedBuildInputs = [
+    chardet
+    attrs
+    commoncode
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = lib.optionals (pythonAtLeast "3.9") [
+    "test_Debian822_from_file__signed_from_dsc"
+    "test_Debian822_from_file__from_status"
+    "test_get_paragraphs_data_from_file__from_status"
+    "test_get_paragraphs_data_from_file__from_sources"
+    "test_get_paragraphs_data_from_file__from_packages"
+    "test_get_paragraphs_data_from_file__from_copyrights_dep5_dropbear"
+    "test_get_paragraphs_data_from_file__from_copyrights_dep5_3"
+    "test_get_paragraphs_data_from_file__from_copyrights_dep5_1"
+    "test_get_paragraph_data_from_file_from_status_can_handle_perl_status"
+    "test_get_paragraph_data_from_file_from_status"
+    "test_get_paragraph_data_from_file__signed_from_dsc_does_not_crash_if_signature_not_removed"
+    "test_get_paragraph_data_from_file__signed_from_dsc_can_remove_signature"
+    "test_DebianCopyright_from_file__from_copyrights_dep5_dropbear"
+    "test_DebianCopyright_from_file__from_copyrights_dep5_3"
+    "test_DebianCopyright_from_file__from_copyrights_dep5_1"
+    "test_parse_contents_ubuntu_with_header_plain"
+    "test_parse_contents_debian_no_header_gzipped"
+  ];
```

Comment:
Why do these only fail on Python 3.9? Did you file an upstream issue?

Reply:
Thanks for reminding me. https://github.com/nexB/debian-inspector/issues/14